pom.xml

```diff
@@ -61,9 +61,9 @@
         <bom>2.289.x</bom>
         <jenkins.version>2.289.1</jenkins.version>
         <changelist>-SNAPSHOT</changelist>
-        <node.version>10.17.0</node.version>
-        <npm.version>6.14.8</npm.version>
-        <frontend-version>1.10.0</frontend-version>
+        <node.version>18.7.0</node.version>
+        <npm.version>8.15.0</npm.version>
+        <frontend-version>1.12.1</frontend-version>
```

Comment:
Upgrade in order to make it pass with Mac M1 chips: https://github.com/eirslett/frontend-maven-plugin/issues/952